generators/app/tasks/appSetup/babelrcSetup.js

```diff
@@ -1,3 +1,23 @@
 module.exports = function babelrcSetup() {
-  this.fs.extendJSON(`${this.projectName}/.babelrc`, { plugins: ['import-glob'] });
+  this.fs.extendJSON(`${this.projectName}/.babelrc`, {
+    plugins: [
+      'import-glob',
+      [
+        'module-resolver',
+        {
+          root: './src',
```

Comment:
I think this must be: `root: ['./src'],`

Reply:
as far as I could find, it's the same thing!